### PR TITLE
Update migration-guides.md

### PR DESCRIPTION
**Pull request by pjfanning**
not perfect but I think this tidies it up a little

### CODE REVIEW

File: docs/src/main/paradox/project/migration-guides.md [hunk 1]
```diff
@@ -8,31 +8,36 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
...
-and @ref:[Cluster](../cluster-usage.md) modules.
-
-It appears that data persisted with "akka-persistence" is usable with "pekko-persistence" (and vice versa).
+### Miscellaneous Notes
 
-We currently do not expect that Akka and Pekko nodes will be able to form a cluster.
+It appears that data persisted with "akka-persistence" is usable with "pekko-persistence" and vice versa (@ref:[Persistence](../persistence.md)).
 
-We may be able to provide [Scalafix](https://scalacenter.github.io/scalafix/) scripts to help with migrations.
+We currently do not expect that Akka and Pekko nodes will be able to form a @ref:[cluster](../cluster-usage.md).
```
Comment (mdedetrich) [inline docs/src/main/paradox/project/migration-guides.md:43]: Can we rephrase this to "We currently do not expect that Akka and Pekko nodes combined will be able to form a" (note the "combined")

Comment (pjfanning) [reply docs/src/main/paradox/project/migration-guides.md]: updated - thanks

File: docs/src/main/paradox/project/migration-guides.md [hunk 2]
```diff
@@ -8,31 +8,36 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
...
 * Pekko packages start with "org.apache.pekko" instead of "akka" - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
 * Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. PekkoException instead of AkkaException
-* Configs in `application.conf` use "pekko" prefix instead of "akka"
+* Config names use "pekko" prefix instead of "akka", e.g. `pekko.actor.provider` instead of `akka.actor.provider`
 * We have changed the default ports used by the pekko-remote module.
 * With @ref:[Classic Remoting](../remoting.md), Akka defaults to 2552, while Pekko defaults to 7355.
 * With @ref:[Artery Remoting](../remoting-artery.md), Akka defaults to 25520, while Pekko defaults to 17355.
+
+### Dependency Changes
 * The Scala 2.13/Scala 3 version of Pekko no longer includes [scala-java8-compat](https://github.com/scala/scala-java8-compat)
```
Comment (mdedetrich) [inline docs/src/main/paradox/project/migration-guides.md:24]: The `version` in "Scala 2.13/Scala 3 version" should be plural

Comment (pjfanning) [reply docs/src/main/paradox/project/migration-guides.md]: added https://github.com/apache/incubator-pekko/pull/397/commits/42373969f416b148537699f4580e09cc3dfe713d

**Review by mdedetrich — APPROVED** [review]
lgtm
